mention_manager.py

```diff
@@ -34,7 +34,7 @@ def parseMentions(self, content, newsgroup):
     line_no = 0
     for raw_line in content.split("\n"):
       line = raw_line.strip()
-      if line.startswith("&gt"):
+      if line.startswith(">"):
```

Comment:
could you also put a space afterwards, i.e. `"> "`.

Reply:
Done!